bin/helpers/sync/syncSpecsLogs.js

```diff
@@ -81,7 +82,7 @@ let printSpecsStatus = (bsConfig, buildDetails) => {
         whileProcess(callback)
       },
       function(err, result) { // when loop ends
-        logger.info("\n--------------------------------------------------------------------------------")
+        noWrap ? logger.info("\n--------------------------------------------------------------------------------") : logger.info("\n" + "-".repeat(terminalWidth))
```

Comment:
can we use `noWrap?` to determine separator length in both cases? Something like
```
logger.info("-".repeat(noWrap ? X : terminalWidth))
```

Reply:
The `X` is default and we would want the existing behaviour to continue in case of a `no-wrap`. We can make it something like:
```node
logger.info("-".repeat(noWrap ? 80 : terminalWidth))
```
Where 80 is the current number of the `-` which are being printed.